### PR TITLE
styling: Remove forVersioning tags in scss

### PR DESCRIPTION
**Pull request by morganm58**
fixes CXSPA-5879
https://jira.tools.sap/browse/CXSPA-5879

### CODE REVIEW

File: feature-libs/cart/base/styles/components/_cart-totals.scss [hunk 1]
```diff
@@ -5,12 +5,8 @@
 
   @include media-breakpoint-down(md) {
     padding-top: 2rem;
-    padding-inline-end: 3rem;
```
Comment (kimhw0630) [inline feature-libs/cart/base/styles/components/_cart-totals.scss]: Did you intentionally remove the following lines?

File: feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss [hunk 2]
```diff
@@ -1,53 +1,29 @@
...
 
-    @include media-breakpoint-up(lg) {
-      margin: 2rem 0 2rem;
-    }
+  @include media-breakpoint-up(lg) {
+    margin: 2rem 0 2rem;
   }
 
   .cx-delivery-mode-wrapper {
-    border: 1px solid var(--cx-color-light);
```
Comment (kimhw0630) [inline feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss:26]: is this for removing `forVersion`?

File: feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss [hunk 3]
```diff
@@ -1,53 +1,29 @@
 %cx-delivery-mode {
   @include checkout-media-style();
 
-  @include media-breakpoint-down(md) {
```
Comment (kimhw0630) [inline feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss:4]: Is this meant for removing 'forVersion'?

File: feature-libs/checkout/base/styles/components/steps/payment/_payment-method.scss [hunk 4]
```diff
@@ -62,18 +56,7 @@
 
   .cx-checkout-text {
     @include media-breakpoint-down(md) {
-      padding-inline-start: 3.5rem;
-
-      @include forVersion(6.3) {
-        padding-inline-start: 0;
-      }
-    }
-    @include media-breakpoint-down(sm) {
```
Comment (kimhw0630) [inline feature-libs/checkout/base/styles/components/steps/payment/_payment-method.scss:71]: Did you intentionally remove this?

File: feature-libs/order/styles/components/order-overview/_order-overview.scss [hunk 5]
```diff
@@ -22,10 +22,8 @@
         flex-grow: 1;
 
         @include media-breakpoint-up(lg) {
-          @include forVersion(1, 6.5) {
-            border-inline-end: 1px solid var(--cx-color-text);
-            margin-inline-end: 10rem;
-          }
+          border-inline-end: 1px solid var(--cx-color-text);
```
Comment (kimhw0630) [inline feature-libs/order/styles/components/order-overview/_order-overview.scss]: forVersion(1, 6.5) means, styles are applied inclusively between versions 1 and 6.5.  do you still need these two lines?

File: feature-libs/cart/base/styles/components/_cart-totals.scss (hunk 1 above)
Comment (morganm58) [reply feature-libs/cart/base/styles/components/_cart-totals.scss:8]: yes

File: feature-libs/checkout/base/styles/components/steps/payment/_payment-method.scss (hunk 4 above)
Comment (morganm58) [reply feature-libs/checkout/base/styles/components/steps/payment/_payment-method.scss:71]: Yes. With the 6.3 version having padding (padding-inline-start: 0;) in both break points we have no need to set the same value twice.

File: feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss (hunk 3 above)
Comment (morganm58) [reply feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss:4]: Yes. i removed the forVersion wrapper and made the scss a little more semantic. I put the media query after the scss for the main selector.

File: feature-libs/cart/base/styles/components/_cart-totals.scss (hunk 1 above)
Comment (morganm58) [reply feature-libs/cart/base/styles/components/_cart-totals.scss]: This one shouldn't be in here. I'll remove it.

File: feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss (hunk 2 above)
Comment (morganm58) [reply feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss:26]: Yes. Some of the styling are duplicated and the scss in the forVersion wrapper overwrites it.

File: feature-libs/order/styles/components/order-overview/_order-overview.scss (hunk 5 above)
Comment (morganm58) [reply feature-libs/order/styles/components/order-overview/_order-overview.scss]: I'll check with Florent about this one. IF the intended use is just for 1, 6.5 to hide the payment info

Comment (morganm58) [reply feature-libs/order/styles/components/order-overview/_order-overview.scss]: its removed now

**Review by kimhw0630 — APPROVED** [review]

**Pull request merged**